docs/src/Badge.doc.js

```diff
@@ -23,6 +23,13 @@ card(
         type: `string`,
         required: true,
       },
+      {
+        name: 'textColor',
+        type: `string`,
+        defaultValue: 'blue',
+        description:
+          'Changes the text color and predefined matching background',
```

Comment:
@denamwangi instead of adding a new prop, could we update the current treatment as specced by Annie?